lib/src/screens/page_d_day_zone_screen/page_d_day_zone_screen.dart

```diff
@@ -0,0 +1,163 @@
+import 'package:easy_localization/easy_localization.dart';
+import 'package:featlink_app/generated/locale_keys.g.dart';
+import 'package:featlink_app/src/config/app_colors.dart';
+import 'package:featlink_app/src/resources/app_assets.dart';
+import 'package:featlink_app/src/screens/page_d_day_zone_screen/widgets/the_d_day_tab.dart';
+import 'package:featlink_app/src/screens/page_d_day_zone_screen/widgets/your_d_day_tab.dart';
+import 'package:flutter/material.dart';
+
+class PageDdayZoneScreen extends StatelessWidget {
```

Comment:
```suggestion
class DDayZoneScreen extends StatelessWidget {
```

Renome ça en **DDayZoneScreen** et le nom du fichier (ainsi que le nom du dossier) en **d_day_zone_screen**